package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "azure-devops-node-api",
     "description": "Node client for Azure DevOps and TFS REST APIs",
-    "version": "12.1.1",
+    "version": "12.2.0",
```

Comment:
Please bump version to 12.3.0 - 12.2.0 is already current version in master

Reply:
Done.